video/transcoder/create_job_from_ad_hoc.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package transcoder
+
+// [START transcoder_create_job_from_ad_hoc]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	transcoder "cloud.google.com/go/video/transcoder/apiv1beta1"
+	transcoderpb "google.golang.org/genproto/googleapis/cloud/video/transcoder/v1beta1"
+)
+
+// createJobFromAdHoc creates a job based on a given configuration. See
+// https://cloud.google.com/transcoder/docs/how-to/jobs#create_jobs_ad_hoc
+// for more information.
+func createJobFromAdHoc(w io.Writer, projectID string, location string, inputUri string, outputUri string) error {
+	// projectID := fmt.Sprintf("my-project-id")
```

Comment:
Is this for documentation/example purposes? Otherwise, please remove.

Another thing we can do is parse the values from flags and set the default values for the flags to serve as an example https://golang.org/pkg/flag/

Reply:
The repo's Go style guidelines say to include these values: https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#function-arguments-for-snippets

You don't need to include parse functionality unless you are doing a quickstart (see https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#function-arguments-for-quickstarts), so I'd prefer to keep these stripped down.

Reply:
+1. Would you mind removing the `fmt.Sprintf` calls? They're just plain strings. I sent #1807 to fix the style guide.

Reply:
Done